packages/@o3r/stylelint-plugin/src/public_api.ts

```diff
@@ -0,0 +1,3 @@
+import rulesPlugins from './index';
+// Rules plugins have to be exported as default
+export default rulesPlugins;
```

Comment:
missing additional empty line

Reply:
in fact there is one 
else we would see the red minus sign in a round